src/cmd/CMakeLists.txt

```diff
@@ -1,17 +1,60 @@
-# Generate a the ruby script.
+#===============================================================================
+# Generate the ruby script for internal testing.
 # Note that the major version of the library is included in the name.
 # Ex: cmdfuel0.rb
-if (APPLE)
-  set(GZ_LIBRARY_NAME lib${PROJECT_NAME_LOWER}.dylib)
-elseif(MSVC)
-  set(GZ_LIBRARY_NAME ${PROJECT_NAME_LOWER}.dll)
-else()
-  set(GZ_LIBRARY_NAME lib${PROJECT_NAME_LOWER}.so)
-endif()
+# Unlike other gz libraries, the ruby script for the fuel_tools library is called cmdfuel.rb instead of cmdfuel_tools.rb
+set(CMD_NAME cmdfuel)
+set(cmd_script_generated_test "${CMAKE_BINARY_DIR}/test/lib/$<CONFIG>/ruby/ignition/${CMD_NAME}${PROJECT_VERSION_MAJOR}.rb")
```

Comment:
ignition -> gz?

Reply:
I amended the merge commit to https://github.com/gazebosim/gz-fuel-tools/pull/363/commits/31bb99766fce404b14fdf1467e5bd1f79950e3c5 with this change